{
  "cveId": "CVE-2025-29088",
  "title": "CVE-2025-29088: In SQLite 3.49.0 before 3.49.1, certain argument values to sqlite3_db_config (in the C-language API)...",
  "description": "In SQLite 3.49.0 before 3.49.1, certain argument values to sqlite3_db_config (in the C-language API) can cause a denial of service (application crash). An sz*nBig multiplication is not cast to a 64-bit integer, and consequently some memory allocations may be incorrect.",
  "publishedDate": "2025-04-10T00:00:00+00:00",
  "lastModifiedDate": "2025-04-14T13:56:32.775000+00:00",
  "severity": "MEDIUM",
  "cvssMetrics": [
    {
      "version": "3.1",
      "vectorString": "CVSS:3.1/AV:L/AC:H/PR:N/UI:N/S:C/C:L/I:L/A:L",
      "baseScore": 5.6,
      "baseSeverity": "MEDIUM"
    }
  ],
  "epss": {
    "score": 0.0005,
    "percentile": 14.51
  },
  "riskScore": 40,
  "exploitationStatus": "UNKNOWN",
  "affectedSystems": {
    "cpeMatches": [],
    "vendors": ["sqlite"],
    "products": ["sqlite"]
  },
  "references": [
    {
      "url": "https://www.sqlite.org/cves.html",
      "source": "",
      "tags": []
    },
    {
      "url": "https://sqlite.org/forum/forumpost/48f365daec",
      "source": "",
      "tags": []
    },
    {
      "url": "https://github.com/sqlite/sqlite/commit/56d2fd008b108109f489339f5fd55212bb50afd4",
      "source": "",
      "tags": []
    },
    {
      "url": "https://gist.github.com/ylwango613/d3883fb9f6ba8a78086356779ce88248",
      "source": "",
      "tags": []
    },
    {
      "url": "https://sqlite.org/releaselog/3_49_1.html",
      "source": "",
      "tags": []
    }
  ],
  "attackTechniques": [],
  "metadata": {
    "tags": ["CWE-190"],
    "requiresUserInteraction": null,
    "requiresPrivileges": null,
    "attackVector": null,
    "sources": [
      {
        "name": "CVEList",
        "url": "https://github.com/CVEProject/cvelistV5/blob/main/cves/2025/29xxx/CVE-2025-29088.json",
        "lastModified": "2025-04-14T13:56:32.775000+00:00"
      }
    ]
  }
}
